Utils/GamePath.cs

```diff
@@ -121,6 +121,11 @@ public static bool ExistsInDataAsPar(string path)
             return FileExistsInData(RemoveModPath(path) + ".par");
         }
 
+        public static bool IsXbox(string path)
+        {
+            return path.Contains("\\Content\\");
```

Comment:
This will only work on Windows. Maybe make it use `Path.DirectorySeparatorChar` instead of a hardcoded `\\`

Reply:
That's unrelated to whether RMM can process the paths correctly or not. RMM should work regardless of the OS you run it on. Please use `DirectorySeparatorChar` instead. Besides, having hardcoded windows-style path separators is not a good idea.